src/components/ReservedUserData/ReservedUserData.tsx

```diff
@@ -0,0 +1,186 @@
+import { ReservedUserDataFragment } from "./ReservedUserDataFragment/ReservedUserDataFragment";
+import { IoIosArrowDown } from "react-icons/io";
+import React, {useContext, useEffect, useState} from "react";
+import {API_URL} from "../../config/apiUrl";
+import {FilterContext} from "../../contexts/filter.context";
+import {ContractType, Internship, TypeWork} from "../../../../HeadHunter_server/types/student/student.enum";
+import {PaginationContext} from "../../contexts/pagination.context";
+import {ReservedStudent} from "../../../../HeadHunter_server/types/student/student.entity";
+
+import "./ReservedUserData.scss";
+import logo from "../../assets/images/avatar-holder.png";
+
+interface Props {
+  id : string;
+  name: string;
+  open :boolean;
+  githubUsername: string;
+  reservationExpiresOn: Date |null
+  FragmentsValues: {
+    header: string;
+    value: string;
+  }[];
+}
+
+type StudentResults = {allRecords: number, data: ReservedStudent[]};
+
+export const ReservedUserData = () => {
```

Comment:
Może lepiej 'ReservedStudent'?

Reply:
można to zmienić ale jakie jest już nazewnictwo wyświetlania wszystkich studentów